### PR TITLE
[djl-serving] Add workflow example

### PR DESCRIPTION
**Pull request by xyang16**

### CODE REVIEW

File: djl-serving/workflows/resnet18/postprocess/model.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                raise Exception(
+                    'labels in mapping must be either str or [str]')
+            mapping[key] = new_value
+        return mapping
+
+
+_service = Processing()
+
+
+def postprocess(inputs: Input) -> Output:
```
Comment (frankfliu) [inline djl-serving/workflows/resnet18/postprocess/model.py:90]: We don't need this function

File: djl-serving/workflows/resnet18/postprocess/model.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    """
+    if not _service.initialized:
+        # stateful model
+        _service.initialize(inputs.get_properties())
+
+    if inputs.is_empty():
+        # initialization request
+        return None
+
+    return postprocess(inputs)
```
Comment (frankfliu) [inline djl-serving/workflows/resnet18/postprocess/model.py:106]:
```suggestion
    return _service.postprocess(inputs)
```

**Review by frankfliu — COMMENTED** [review]
Add link to here: https://github.com/deepjavalibrary/djl-demo/tree/master/development/python#use-djl-serving-workflow

File: djl-serving/workflows/resnet18/postprocess/model.py (hunk 1 above)
Comment (xyang16) [reply djl-serving/workflows/resnet18/postprocess/model.py]: Changed

**Comment by xyang16** [comment]
> Add link to here: https://github.com/deepjavalibrary/djl-demo/tree/master/development/python#use-djl-serving-workflow

Added

**Review by frankfliu — APPROVED** [review]

**Pull request merged**